projects/coredns/coredns/Makefile

```diff
@@ -31,7 +31,8 @@ IMAGE_NAME?=coredns/coredns
 # This tag is overwritten in the prow job to point to the upstream git tag and this repo's commit hash
 IMAGE_TAG?=${GIT_TAG}-eks-${RELEASE_BRANCH}-${RELEASE}
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
-
+DATE_EPOCH_TAG=$(shell date "+%F-%s")
```

Comment:
What about something like `git show --no-patch --no-notes --pretty='%ct'  $SHA1` so we get the same date for all the containers. Now that I say that though, maybe just tagging them all with the $SHA1 would be easier and more easy to correlate.